site/pages/docs/components/custom/thumbnail.vue

```diff
@@ -0,0 +1,167 @@
+<template>
+  <div class="docs__section">
+    <div class="docs__header">
+      <h1>Thumbnail</h1>
```

Comment:
i'm leaning towards having an `image` component with `--thumbnail` style

versus having a dedicated `thumbnail` component that will need to do exactly what an image component would

Reply:
Lets rename it to `.image` and have a default style of `max-width: 100%`? Then add the modifier `--thumbnail` yeah. It would make much more sense, since `image` and `thumbnail` wouldn't be a separate component later on.